README.md

```diff
@@ -1 +1,13 @@
-DataHub documentation and blog
\ No newline at end of file
+<p align="center">
+  <a href="https://datahub.io/">
+    <img alt="datahub" src="http://datahub.io/static/img/logo-cube.png" width="146">
+  </a>
+</p>
+
+<p align="center">
+  This is a repository for DataHub documentation - https://datahub.io/docs
+  <br />
+  <br /><a href="https://gitter.im/datahubio/chat"><img src="https://img.shields.io/gitter/room/frictionlessdata/chat.svg" /></a>
+</p>
+
+Please feel free to open a pull requests, if there's anything you feel we could do better!
```

Comment:
Either `a pull request` (singular) or `pull requests` (without article `a`).
Also, instead of `could do better` I would have `could improve`.